scripts/uninstall.sh

```
@@ -38,19 +38,19 @@ fi
 print_status "${INFO}Proceeding with uninstalling reconPoint${RESET}"
 
 print_status "Stopping all containers related to reconPoint..."
-docker stop $(docker ps -a -q --filter name=reconpoint) 2>/dev/null
+docker stop $(docker ps -a -q --filter name=reconpoint-) 2>/dev/null
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)